Lesson34/Merve/Lesson33/props-drilling-app/src/modules/user/Account/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { UserDetailsForm } from '../UserDetailsForm';
+import { useContext, useState } from 'react';
+import { UserContext, UserDispatchContext } from '../UserProvider';
+
+export const Account = () => {
+  const user = useContext(UserContext);
+  const setUser = useContext(UserDispatchContext);
+  const [showDetails, setShowDetails] = useState(true);
+
+
+  const handleNameChange = (newName) => {
```

Comment:
Account component should not have logic for updating user data, they belong to the UserDetailsForm component.
Remove the following:
- `  const setUser = useContext(UserDispatchContext);`
- handleNameChange function
- onNameChange prop passed to UserDetailsForm component